tests/test_core.py

```diff
@@ -1149,23 +1149,23 @@ def test_group(self, moco_zgs):
         assert isinstance(moco_zgs.group, ZoneGroup)
         assert moco_zgs in moco_zgs.group
 
-    def test_all_zones(selfself, moco_zgs):
+    def test_all_zones(self, moco_zgs):
```

Comment:
I think these other methods were as they were, with the `selfself` first argument to prevent some linting issue. It may be possible to handle that in another way, but let's do that separately. So please remove this and the next three single line changes that are not associated with tests for the new feature.